src/homeform.cpp

```diff
@@ -208,7 +208,7 @@ homeform::homeform(QQmlApplicationEngine *engine, bluetooth *bl) {
     steeringAngle = new DataObject(QStringLiteral("Steering"), QStringLiteral("icons/icons/cadence.png"),
                                    QStringLiteral("0"), false, QStringLiteral("steeringangle"), 48, labelFontSize);
     peloton_offset =
-        new DataObject(QStringLiteral("Peloton Offset"), QStringLiteral("icons/icons/clock.png"), QStringLiteral("0"),
+        new DataObject(QStringLiteral("Peloton Rem."), QStringLiteral("icons/icons/clock.png"), QStringLiteral("0"),
```

Comment:
we can't change tile in this way, you have to keep in mind that thousand of users are you using this. Changing even something very little cause a lot of work for me to inform all the users. I would like to revert this.